src/systemcmds/sd_bench/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@
 px4_add_module(
 	MODULE systemcmds__sd_bench
 	MAIN sd_bench
+	STACK_MAIN 1500
```

Comment:
Why? This is smaller than the default.

Reply:
In debugging hardfaults caused by the AIX errata I believe I did a stack check build and set it to what it needed.  We can revert these to use the default

Reply:
Looks like that happened before the default stack main was bumped in master.